google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITAddressesTest.java

```diff
@@ -120,4 +103,36 @@ public void testAggregatedList() {
     }
     Assert.assertTrue(presented);
   }
+
+  @Ignore("Non ascii symbols are not converted correctly")
+  @Test
+  public void testNonAscii() {
+    insertAddress("тест");
+    Address address = addressesClient.get(DEFAULT_PROJECT, DEFAULT_REGION, name);
+    Assert.assertEquals(name, address.getName());
+    Assert.assertEquals("тест", address.getDescription());
+  }
+
+  private void insertAddress() {
+    insertAddress("test");
+  }
+
+  private void insertAddress(String description) {
+    Address address = Address.newBuilder().setName(name).setDescription(description).build();
+    Operation operation = addressesClient.insert(DEFAULT_PROJECT, DEFAULT_REGION, address);
+    addresses.add(address);
+    boolean success = false;
+    long startTime = System.currentTimeMillis();
+    while ((System.currentTimeMillis() - startTime) < 15000) {
+      Operation op =
+          regionOperationsClient.get(DEFAULT_PROJECT, DEFAULT_REGION, operation.getName());
```

Comment:
Please always add at least `Thread.sleep()` between in polling loops (as the simplest delay mechanism you can employ). Check the other comment for details why it is important. `Thread.sleep(1000)` for example (so the `regionOperationsClient.get()` will be performed each second, instead of being performed immediately after the previous call, basically "DDoS'ing" the server)